### ISSUE TITLE
Error: Failed to find a python interpreter in the .data section

### ISSUE THREAD

**Issue by OM152** [benfred/py-spy#684]
* Windows 11
* Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)] on win32
* py-spy==0.3.14
* PyCharm 2024.1 (Community Edition) Build #PC-241.14494.241, built on March 28, 2024 Runtime version: 17.0.10+8-b1207.12 amd64

Hi @benfred,
I'm trying to run py-spy.

1. Run my script
```
(.venv) PS W:\PycharmProjects\production_counters_master_2> W:\PycharmProjects\production_counters_master_2\.venv\Scripts\python.exe -m run_prod_counter
```
or 
`(.venv) PS W:\PycharmProjects\production_counters_master_2> python run_prod_counter.py`
Script is running.

2. then I run
`py-spy record -o output.svg --pid 7156`
or
`py-spy top --pid 7156`

3. I get an error
'Error: Failed to find a python interpreter in the .data section'

Can you figure out this problem? Thanks

**Comment by zanieb** [comment]
Is this a duplicate of https://github.com/benfred/py-spy/issues/675

**Comment by benfred** [comment]
This should be fixed by https://github.com/benfred/py-spy/pull/718

**Comment by benfred** [comment]
I think this should be fixed in the 0.4 release. Can you try it out and let me know? 

If this is still not working please re-open the issue
